### PR TITLE
feat: update forma36 packages [EXT-5594]

### PR DESCRIPTION
**Pull request by sarahlessner**
## Purpose

Updating the forma 36 packages to the latest versions

## Testing steps

Comparing the testing version for development against the current version of the sap testing site to see if there was any CSS issues.

### CODE REVIEW

File: apps/sap-commerce-cloud/src/components/Dialog/Dialog.styles.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { css } from 'emotion';
-import tokens from '@contentful/forma-36-tokens';
+import tokens from '@contentful/f36-tokens';
 
 export const styles = {
   textInput: css({ width: '250px' }),
-  grid: css({ marginTop: tokens.spacingM, paddingLeft: tokens.spacingL }),
+  grid: css({ margin: `${tokens.spacingM} 0`, paddingLeft: tokens.spacingL }),
```
Comment (mgoudy91) [inline apps/sap-commerce-cloud/src/components/Dialog/Dialog.styles.ts:6]: This looks like a style change, was it intentional?

**Review by mgoudy91 — COMMENTED** [review]
Looks good so far! Happy to re-review and approve when the last part gets in!

File: apps/sap-commerce-cloud/src/components/Dialog/Dialog.styles.ts (hunk 1 above)
Comment (sarahlessner) [reply apps/sap-commerce-cloud/src/components/Dialog/Dialog.styles.ts:6]: Yep. When reviewing the portions of the app we updated, we noticed that there was some spacing built into the card before that isn't there with the new component library. We needed to have this margin apply to the top and bottom now that the element below this input section doesn't have top spacing.

File: apps/sap-commerce-cloud/package.json [hunk 2]
```diff
@@ -6,11 +6,12 @@
   "dependencies": {
     "@contentful/app-scripts": "^1.7.13",
     "@contentful/app-sdk": "^4.23.0",
+    "@contentful/f36-button": "^4.68.1",
```
Comment (whitelisab) [inline apps/sap-commerce-cloud/package.json:9]: Curious if it makes sense to remove this dependency and just use the button from the `@contentful/f36-components` package?

**Review by whitelisab — APPROVED** [review]
One small comment/question, otherwise looks awesome. Great work! 🌟

File: apps/sap-commerce-cloud/package.json (hunk 2 above)
Comment (sarahlessner) [reply apps/sap-commerce-cloud/package.json:9]: I saw an example in the migration guide for the Icon Button that imported it from there so I just didn't realize I could get it from the main component library as well. I agree that makes sense then and I'll make that change.

**Pull request merged**